wiki/en/Running-a-Server.md

```diff
@@ -50,55 +49,47 @@ Unless you plan on hosting more than about 5 players on a slower-speed home conn
-	<figcaption>How Private Servers work</figcaption>
+	<img src="{% include img/en-screenshots/diagram-unreg-server.inc %}" loading="lazy" alt="Diagram of connections between clients within a Jamulus Unregistered Server">
+	<figcaption>How Unregistered Servers work</figcaption>
 </figure>
 
+### 3. Custom Directory
 
+If you want to run a number of Servers, possibly also behind a firewall or on a LAN, you may want to run your server as a Directory. Examples include online events, music associations, sectional rehearsals or music lessons for schools.
 
-### 3. Directory
-
-For when you want to run a number of Private Servers, possibly also behind a firewall or on a LAN. Examples include online events, music associations, sectional rehearsals or music lessons for schools.
-
-To view servers listed by a custom Directory Server, musicians must enter the address in their client’s "Custom Directory Server" settings field.  They will then see a Connection Setup list which is generated by your Directory.
+To view servers listed by a Custom Directory, musicians must enter its address in their client’s "Custom Directory Server" settings field.  They will then see a Connection Setup list which is generated by your Directory. Custom Directories otherwise work for clients in the same way as Public Directories.
 
 Up to 150 servers can then register with your Directory by setting your Directory address in their `--directoryserver` option at startup.
 
-To run a server as a Directory, it should be configured with `--directoryserver localhost` (that is, making itself the Directory to query for servers).
+To run a server as a Directory, it should be configured with `--directoryserver localhost` (that is, making itself the Directory to query for servers). When running a server with the GUI, select "Custom" from the Directory drop-down list. 
 
-#### Points to note about Directories
+#### Points to note about Custom Directories
 
 - If you want to control which servers can register with your Directory, you can enable a whitelist with the `--listfilter` command line option. See the section on command line options later in this document.
 
-- When running a public Directory Server behind a NAT firewall on a private network, use the `--serverpublicip` option to specify the public IP address of the server(s) being listed by your Directory. This is necessary to allow clients on the public Internet to connect to them via NAT. Note that for the servers using this option, you will still need proper port forwarding in your router/firewall.
+- When running a Directory behind a NAT firewall on a private network, use the `--serverpublicip` option to specify the public IP address of the server(s) being listed by your Directory. This is necessary to allow clients on the public Internet to connect to them via NAT. Note that for the servers using this option, you will still need proper port forwarding in your router/firewall.
```

Comment:
>  public IP address of the server(s) being listed by your Directory

Only applies to servers on the same LAN / with the same public IP.  The phrasing implies it applies to _all_ registered servers.